### PR TITLE
py-tornado: add 6.1

### PR DESCRIPTION
**Pull request by manuelakuhn**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-tornado/package.py [hunk 1]
```diff
@@ -10,19 +10,22 @@ class PyTornado(PythonPackage):
...
-    version('6.0.3',   sha256='a97ac3b8c95867e534b48cb6fbbf156f5ca5b20c423bb06894c17b240d7a18fc')
-    version('5.1.1', sha256='a1da335a2978b9a8c3544cab10076d799442d7988ed0b4f2be035fe4388ca8dd')
-    version('4.4.0', sha256='ae556a0848e5d428d00597a18b38b9ca9d20f4600535e1dd33b3a576ab234194')
+    version('6.1', sha256='33c6e81d7bd55b468d2e793517c909b139960b6c790a60b7991b9b6b76fb9791')
+    version('6.0.3', sha256='a97ac3b8c95867e534b48cb6fbbf156f5ca5b20c423bb06894c17b240d7a18fc',
+            url="https://github.com/tornadoweb/tornado/archive/v6.0.3.tar.gz")
+    version('5.1.1', sha256='a1da335a2978b9a8c3544cab10076d799442d7988ed0b4f2be035fe4388ca8dd',
+            url="https://github.com/tornadoweb/tornado/archive/v5.1.1.tar.gz")
+    version('4.4.0', sha256='ae556a0848e5d428d00597a18b38b9ca9d20f4600535e1dd33b3a576ab234194',
+            url="https://github.com/tornadoweb/tornado/archive/v4.4.0.tar.gz")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tornado/package.py]: Let's just download the old versions from pypi too. You'll need to change the sha256s.

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-tornado/package.py]: I can do that but on pypi version `4.4.0` is called only `4.4`. Is it an issue if I rename the version number to 4.4 or should I set an explicit url for this version?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-tornado/package.py]: Hmm, you could have both 4.4 and 4.4.0 (and explicitly override the url for 4.4.0) and deprecate 4.4.0 if you want to be careful, but I would be okay with just renaming the version.

Comment (manuelakuhn) [reply var/spack/repos/builtin/packages/py-tornado/package.py]: Ok, I checked and did not find any package inside of spack which has an explicit dependency to 4.4.0, so I guess it is ok to drop it.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**